.gitignore

```diff
@@ -8,3 +8,4 @@ static/js/app.js
 openfecwebapp/local_config.py
 static/styles/styles.css.map
 app/
+/.env
```

Comment:
No complaints here, but kind of thing can also go in a global .gitignore, since not everyone uses autoenv. Even though it's great.